### PR TITLE
[DOC] Typescript SDK overview page

### PR DESCRIPTION
**Pull request by msavir**
### Description
Updates Typescript SDK overview page

### CODE REVIEW

File: developer-docs-site/docs/sdks/ts-sdk/typescript-sdk-overview.md [hunk 1]
```diff
@@ -63,18 +63,27 @@ The OpenAPI client is a set of classes that are generated based on the Aptos RES
 
 ### Aptos Account
 
-The [class AptosAccount](https://aptos-labs.github.io/ts-sdk-doc/classes/AptosAccount.html) provides the methods for:
+The [class AptosAccount](https://aptos-labs.github.io/ts-sdk-doc/classes/AptosAccount.html) has a constructor that creates a new account instance or retrives an existing account instance, in addition it provides the methods for:
```
Comment (movekevin) [inline developer-docs-site/docs/sdks/ts-sdk/typescript-sdk-overview.md:66]: nit: typo retrives -> retrieves

Comment (movekevin) [inline developer-docs-site/docs/sdks/ts-sdk/typescript-sdk-overview.md:66]: Also to keep sentences short => "an existing account instance. In addition, ..."

File: developer-docs-site/docs/sdks/ts-sdk/typescript-sdk-overview.md [hunk 2]
```diff
@@ -63,18 +63,27 @@ The OpenAPI client is a set of classes that are generated based on the Aptos RES
...
 
 - Generating Ed25519 key pairs.
 - Signing a bytes buffer with an Ed25519 public key, and
 - Deriving initial account addresses from the public keys.
+- Retrives a resource account address by source address and seeds
```
Comment (movekevin) [inline developer-docs-site/docs/sdks/ts-sdk/typescript-sdk-overview.md:71]: nit: Retrives => Retrieves

**Review by movekevin — APPROVED** [review]
Nice!

File: developer-docs-site/docs/sdks/ts-sdk/typescript-sdk-overview.md (hunk 1 above)
Comment (clay-aptos) [reply developer-docs-site/docs/sdks/ts-sdk/typescript-sdk-overview.md:66]: I got this. :-)

Comment (clay-aptos) [reply developer-docs-site/docs/sdks/ts-sdk/typescript-sdk-overview.md:66]: Done!

**Review by clay-aptos — APPROVED** [review]
Thank you so much for the wonderful additions, Maayan! Great work. :-)

**Pull request merged**